scripts/west_commands/runners/intel_adsp.py

```diff
@@ -68,9 +73,15 @@ def do_add_parser(cls, parser):
                             help='the default basename of the key store in board.cmake')
         parser.add_argument('--key',
                             help='specify where the signing key is')
-        parser.add_argument('--pty', action="store_true",
-                            help='the log will not output immediately to STDOUT, you \
-                            can redirect it to a serial PTY')
+        parser.add_argument('--pty',
+                            help='the log will be output immediately to STDOUT, the \
```

Comment:
```suggestion
                            help='Capture the output of cavstool.py running on --remote-host and stream it remotely to west's standard output
```

BTW cavstool.py uses both stdout and stderr to print two different things. How are they forwarded here?

Reply:
That's indeed a problem now. Currently the log from stderr output to log at server-side. How do you think that forwarding them to log service also?

Reply:
@enjiamai I'm not sure if this comment is still open or not, but I'm going to leave it up to you to decide if this current implementation is OK as I don't know anything about cavstool.py